vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/GridMultiSelectionModel.java

```diff
@@ -41,13 +41,17 @@
      * only visible if an in-memory data provider is used
      * {@link DataProvider#isInMemory()}.
      */
-    public enum SelectAllCheckboxVisibility {
+    enum SelectAllCheckboxVisibility {
```

Comment:
These have been public for a reason - don't force people to think by removing the public parameter

Reply:
Reverted